ubatch/data_request.py

```diff
@@ -192,7 +195,24 @@ def full(self) -> bool:
         return len(self) >= self._size
 
     def get_inputs(self) -> List[T]:
```

Comment:
I would rather separate this into two methods:

1. get_args_inputs()
2. get_kwargs_inputs()